css/components.css

```
@@ -20,50 +20,267 @@
+  font-size: var(--number-size);
   font-weight: 700;
-  font-size: 2.5rem;
   color: var(--primary-color);
-  letter-spacing: 1px;
-  margin-bottom: 8px;
+  letter-spacing: 0.5px;
+  margin-bottom: var(--number-margin);
+  line-height: 1.1;
+  word-wrap: break-word;
+  overflow-wrap: break-word;
+  font-size: clamp(1.5rem, 4vw, var(--number-size));
```

Biome (2.1.2)

[error] Duplicate properties can lead to unexpected behavior and may override previous declarations unintentionally.

font-size is already defined here.

Remove or rename the duplicate property to ensure consistent styling.

(lint/suspicious/noDuplicateProperties)